### PR TITLE
chore(1747) - SPAship UI enhancements for SPA List page and SPA details page

### LINKED ISSUE #1747: [Chore]: SPAship UI enhancements for SPA List page and SPA details page
## Describe the solution you'd like:

- https://xd.adobe.com/view/707e959d-a616-46af-abc3-ed9723d6ea99-252d/screen/9f4a6575-ce40-4310-8a3b-1b5a606bb923/
- 
- https://xd.adobe.com/view/6920567e-0b80-4122-a579-43197d545c22-3c73/screen/6b9dd8fa-1703-4e40-bb4a-1a1b1a2d0f35/
- 
## Describe alternatives you've considered:

- None

### PR DESCRIPTION
**Pull request by hybridx**
## Closes / Fixes / Resolves

- Resolves #1747 

## Explain the feature/fix

- Minor UI fixes, Added table view for SPAList

## Does this PR introduce a breaking change

- No

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

Screenshot 2022-05-24 at 3 20 11 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/components/layout/header/index.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const Header: FunctionComponent<HeaderProps> = ({ breadcrumbs = [], buttons = []
                   }}
                 >
                   <Label icon={<CogsIcon />}>
-                    <span>Settings/Environments</span>
+                    <span>Environment config</span>
```
Comment (sayak-sarkar) [inline packages/manager/components/layout/header/index.tsx:81]:
```suggestion
                    <span>Environment Configuration</span>
```

File: packages/manager/pages/properties/[propertyName]/index.tsx [hunk 2]
```diff
@@ -73,49 +75,49 @@ const StyledText = styled(Text)`
...
             ]
         );
-        const [activitesResponse, countResponse]: AnyProps = response;
-        if (activitesResponse == null || countResponse == null) {
-            return { props: { url: getGuideUrl() } };
-        }
+        const [activitiesResponse, countResponse]: AnyProps = response;
+        // if (!!activitiesResponse?.length || !!countResponse?.length) {
+        //     return { props: { url: getGuideUrl() } };
+        // }
```
Comment (sayak-sarkar) [inline packages/manager/pages/properties/[propertyName]/index.tsx:102]: Remove commented code!

**Review by sayak-sarkar — APPROVED** [review]
Mostly looks good to me. Suggested a few minor changes.

File: packages/manager/components/layout/header/index.tsx (hunk 1 above)
Comment (hybridx) [reply packages/manager/components/layout/header/index.tsx:81]: This makes sense

File: packages/manager/pages/properties/[propertyName]/index.tsx (hunk 2 above)
Comment (hybridx) [reply packages/manager/pages/properties/[propertyName]/index.tsx:102]: Sorry it got commited, I'm still working on this and it was committed 😅

**Review by soumyadip007 — APPROVED** [review]

**Comment by soumyadip007** [comment]
Looks good to me

**Pull request merged**